### PR TITLE
Changes robot maximum speed limit

### PR DESCRIPTION
**Pull request by joaocarloscampi**
Defined at the meeting on April 20th, robots will have maximum velocity limitation to 1.5m/s.

### CODE REVIEW

**Review by GiovannaNascimento42 — APPROVED** [review]

**Comment by GiovannaNascimento42** [comment]
Forgot to leave a comment on the approval, but the changes are OK.

File: chapters/offenses.adoc [hunk 1]
```diff
@@ -126,6 +126,8 @@ the violation before being penalized additional times.
 
 ===== Attacker Touched Ball In Opponent Defense Area
 The ball must not be touched by a robot, while the robot is partially or fully inside the opponent <<Defense Area, defense area>>.
+===== Robot Speed
+The robot must not exceed a speed of 1.5 meters per second.
```
Comment (ersaraujo) [inline chapters/offenses.adoc]:
```suggestion
The ball must not be touched by a robot, while the robot is partially or fully inside the opponent <<Defense Area, defense area>>.

===== Robot Speed
The robot must not exceed a speed of 1.5 meters per second.
```

Comment (ersaraujo) [reply chapters/offenses.adoc]: Added a line between topics. Otherwise, the visualization will be broken.

**Review by ersaraujo — CHANGES_REQUESTED** [review]

File: chapters/offenses.adoc (hunk 1 above)
Comment (ersaraujo) [reply chapters/offenses.adoc]: @joaocarloscampi can you check this?

**Review by ersaraujo — APPROVED** [review]

**Pull request merged**